ancar/portfolio/src/main/webapp/style.css

```diff
@@ -222,23 +226,39 @@ p {
 }
 
 #fourth-row {
+    padding-top: 50px;
+    padding-bottom: 50px;
```

Comment:
again, this is big enough that you want it to adopt to the place we have.

Reply:
Done in new commit :)